relnotes/reopensocket.feature.md

```diff
@@ -0,0 +1,20 @@
+* `ocean.util.app.DaemonApp`
+
+  `DaemonApp`'s `ReopenableFilesExt` now supports reopening commands from the
+  UnixSocketExt. If you have the UnixSocketExt bound to the socket, sending
+  command `reopen_logfile file_path` will reopen the specified file, if
+  registered, and acknowledge the action with `ACK` or `ERROR` strings.
+
+  With this in place, reopening only affected files (vs all open registered files)
+  with logrotate can be done as follows:
+
+    1) use `nosharedscripts` directive, so the post-rotate script is called
+       once per file, with the absolute file path passed to it.
+    2) From the absolute file path, get the log directory and deduce socket
+       location from it (application could place the socket in its `run/`
+       directory, so the socket path would be
+       `SOCKETPATH=$(dirname $LOGPATH)/../application.socket`.
+    3) Finally, in post-rotate script, simply executing
+       `test $(echo reopen_logfile $LOGPATH | nc -q 2 -U $SOCKETPATH) = ACK` will
```

Comment:
You probably should add quotes to `$(echo ...)`, otherwise it could be an empty string and make `test` give an error, right?

Reply:
Yeah.

Reply:
Good catch!